docs/guide/areas-and-widgets/custom-widgets.md

```diff
@@ -204,3 +204,42 @@ apos.util.widgetPlayers.accordion = {
 ```
 
 [Credit goes to Heydon Pickering](https://inclusive-components.design/collapsible-sections/) for the accessible collapser example.
+
+### Using widget data in players
+
+These widget players do not know anything about the data in the widget. If we want to use widget data in your player code, we will need to provide it.
```

Comment:
This is phrased a bit strangly, but I'm also hesitant to provide a suggestion. This is what I think its saying.

```suggestion
By default, widget players don't have access to widget data. Widget data must be explicitly provided to players in order to utilize it.
```